controllers/workspace/http.go

```diff
@@ -70,7 +70,7 @@ func setupHttpClients(k8s client.Client, logger logr.Logger) {
 	}
 	healthCheckHttpClient = &http.Client{
 		Transport: healthCheckTransport,
-		Timeout:   500 * time.Millisecond,
+		Timeout:   3 * 500 * time.Millisecond,
```

Comment:
This would actually be 1500 ms (1.5s) instead of 3s.

Reply:
Without a doubt a checkout of a stash to high, the test that I deployed was set to a hard coded 3s